scripts/smoke_test_deploy.sh

```
@@ -0,0 +1,118 @@
+
+# Build the frontend image
+echo "── Building frontend image ──"
+docker build -t sat-smoke-frontend -f frontend/Dockerfile frontend/
+echo ""
+
+# Start container (API_KEY not needed for static asset tests)
+docker run -d --name "$CONTAINER_NAME" -p "$PORT:80" \
+    -e API_KEY=smoke-test-key sat-smoke-frontend >/dev/null
+
+# Wait for nginx to be ready
+for i in $(seq 1 10); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)